spec/models/like_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'Like Model Test' do
+    subject { Like.new(author_id: 3, post_id: 2) }
+    before { subject.save }
+
+    it 'The author_id should be number' do
```

Comment:
- [ ] Good job so far :clap: But one of your tests is falling :cry:  Please check the screenshot below :arrow_down: to see the errors and fix them to make all your tests pass as expected :+1: 
![Screenshot_20230330_222931](https://user-images.githubusercontent.com/6615394/228978846-4bcbe4d2-00f2-48f4-a703-30f6b9e20213.jpg)

Reply:
Hi @codecaiine and thanks for the review.
Here is the new test result:

<img width="1002" alt="Screenshot 2023-03-31 102805" src="https://user-images.githubusercontent.com/58295348/229053147-ed4e577a-c25b-45b3-85e1-af337861fe4f.png">